src/test/java/com/wikia/webdriver/common/contentpatterns/AdSlot.java

```diff
@@ -3,27 +3,31 @@
 import lombok.Getter;
 
 public enum AdSlot {
-  TOP_LEADERBOARD("top_leaderboard","div[id*='wka1b.LB/top_leaderboard'][id*='_0__container__']"),
-  BOTTOM_LEADERBOARD("bottom_leaderboard","div[id*='wka1b.PF/bottom_leaderboard'][id*='_0__container__']"),
-  TOP_BOXAD("top_boxad", "div[id*='wka1b.MR/top_boxad'][id*='_0__container__']"),
-  INCONTENT_BOXAD("incontent_boxad", "div[id*='/incontent_boxad']"),
-  BOTTOM_BOXAD("bottom_boxad", "div[id$='bottom_boxad_0__container__']"),
-  FEED_BOXAD("feed_boxad", "div[id$='google_ads_iframe_/5441/wka1b.PF/feed_boxad/desktop/ns-article/_fandom-all_0__container__']"),
-  FEED_BOXAD_MOBILE("feed_boxad", "div[id$='google_ads_iframe_/5441/wka1b.PF/feed_boxad/smartphone/ns-article/_fandom-all_0__container__']"),
-
-  FEED_BOXAD_IFRAME("feed_boxad", "div[id$='google_ads_iframe_/5441/wka1b.PF/feed_boxad/desktop/ns-topic/_fandom-all_0__container__']"),
-  FEED_BOXAD_MOBILE_IFRAME("feed_boxad", "div[id$='google_ads_iframe_/5441/wka1b.PF/feed_boxad/smartphone/ns-topic/_fandom-all_0__container__']"),
-  BOTTOM_LEADERBOARD_IFRAME("bottom_leaderboard", "div[id*='wka1b.PF/bottom_leaderboard'][id*='_0__container__'] iframe");
+  TOP_LEADERBOARD("top_leaderboard","div[id*='wka1b.LB/top_leaderboard'][id*='_0__container__']", "div[id*='wka1b.LB/top_leaderboard']"),
+  BOTTOM_LEADERBOARD("bottom_leaderboard","div[id*='wka1b.PF/bottom_leaderboard'][id*='_0__container__']","div[id*='wka1b.PF/bottom_leaderboard']"),
+  TOP_BOXAD("top_boxad", "div[id*='wka1b.MR/top_boxad'][id*='_0__container__']", "div[id*='wka1b.MR/top_boxad']"),
+  INCONTENT_BOXAD("incontent_boxad", "div[id*='/incontent_boxad']", "div[id*='/incontent_boxad']"),
+  BOTTOM_BOXAD("bottom_boxad", "div[id$='bottom_boxad_0__container__']", "#bottom_boxad"),
+  FEED_BOXAD("feed_boxad", "div[id$='google_ads_iframe_/5441/wka1b.PF/feed_boxad/desktop/ns-article/_fandom-all_0__container__']", "#feed_boxad"),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "#feed_boxad" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "feed_boxad" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)